File: VB/CustomSchedulerEvent.Module.Web/WebModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel

Imports DevExpress.ExpressApp

Namespace CustomSchedulerEvent.Module.Web
	<ToolboxItemFilter("Xaf.Platform.Web")> _
	Public NotInheritable Partial Class CustomSchedulerEventAspNetModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
	End Class
End Namespace

File: CS/CustomSchedulerEvent.Module.Web/WebModule.cs
using System;
using System.ComponentModel;

using DevExpress.ExpressApp;

namespace CustomSchedulerEvent.Module.Web {
    [ToolboxItemFilter("Xaf.Platform.Web")]
    public sealed partial class CustomSchedulerEventAspNetModule : ModuleBase {
        public CustomSchedulerEventAspNetModule() {
            InitializeComponent();
        }
    }
}
